src/components/20-molecules/datepicker/ui.test.js

```diff
@@ -389,6 +373,25 @@ test('should highlight today', async t => {
   await t.expect(cell).eql('TODAY');
 });
 
+fixture('Datepicker without a valid startyear')
+  .page(
+    `${host}/iframe.html?id=components-datepicker--datepicker&knob-locale=de-CH&knob-year=x&knob-month=4&knob-day=22&knob-allowedyears=%5B%221971-2000%22%2C2012%2C2014%2C%222018-2022%22%5D&knob-label=&knob-monthtitle=Choose%20Month&knob-yeartitle=Choose%20Year&knob-invaliddatetext=Invalid%20date&knob-placeholder=Please%20select%20a%20date&knob-margin-top=0&knob-width%20%28not%20an%20attribute%29=&knob-disabled=true&viewMode=story`
```

Comment:
very hard to see: knob-year=x. Either add a comment or use the power of string template literals to do:
`const ILLEGAL_YEAR = 'x'; .... &knob-year=${ILLEGAL_YEAR}...`

Reply:
das problem bei dem Test ist, dass ich eig. gar kein year setzen will
aber das lässt storybook irgendwie nicht zu